packages/config/config/devices/0x0330/zv2835rac-nf.json

```
@@ -0,0 +1,273 @@
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV2835RAC-NF",
+	"description": "Knob Smart Dimmer",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd017"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"2": {
+			"label": "State before power failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Turn off load",
+					"value": 0
+				},
+				{
+					"label": "Turn on load",
+					"value": 1
+				},
+				{
+					"label": "Save load state before power failure",
+					"value": 2
+				}
+			]
```

This comes up often enough that we should make two master templates for it. One with "previous state", one without.

IMO, this is the preferred definition (short and clear):
```suggestion
			"label": "State After Power Failure",
			"valueSize": 1,
			"minValue": 0,
			"maxValue": 2,
			"defaultValue": 2,
			"allowManualEntry": false,
			"options": [
				{
					"label": "Off",
					"value": 0
				},
				{
					"label": "On",
					"value": 1
				},
				{
					"label": "Previous State",
					"value": 2
				}
			]
```

---

Once #2665 is merged, the preferred one would be
```suggestion
"$import": "../templates/master_template.json#state_after_power_failure_off_on_prev"
```